tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala

```diff
@@ -82,6 +85,11 @@ class SQLPlanParserSuite extends FunSuite with BeforeAndAfterEach with Logging {
           assert(projects.forall(_.isSupported == true))
           assert(projects.forall(_.children.isEmpty))
           assert(projects.forall(_.duration.isEmpty))
+          val sorts = allChildren.filter(_.exec == "SortExec")
```

Comment:
is it called SortExec or Sort?  The tests above for FilterExec and SortExec actually have a bug in them.
Add a check to make sure the sorts.size is 1

Reply:
Thanks for catching this. Updated the test. There are 3 Sorts in the the plan.